modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -256,10 +263,9 @@ ALTER USER name [IF EXISTS]
   [[SET PASSWORD] CHANGE [NOT] REQUIRED]
   [SET STATUS {ACTIVE \| SUSPENDED} ]
   [SET HOME DATABASE name]
-  [SET AUTH [PROVIDER] 'provider' "{"SET <key> <value>"}"]
+  {[SET AUTH [PROVIDER] 'provider' "{"{SET <key> <value>}..."}"]}...
   [REMOVE HOME DATABASE]
-  [REMOVE { AUTH [PROVIDER[S]] provider[, ...] } \| { ALL AUTH [PROVIDER[S]] }]
-
+  [REMOVE { AUTH [PROVIDER[S]] provider[, ...] \| ALL AUTH [PROVIDER[S]] }]
```

Comment:
the remove auth should also allow multiple
```suggestion
  [REMOVE { AUTH [PROVIDER[S]] provider[, ...] \| ALL AUTH [PROVIDER[S]] }]...
```
or
```suggestion
  {[REMOVE { AUTH [PROVIDER[S]] provider[, ...] \| ALL AUTH [PROVIDER[S]] }]}...
```
if we want the extra brackets that SET currently have

Reply:
this of course still don't show you can have both REMOVE AUTH clauses but that might be ok as it won't give the user anything to add both 🤷